src/Files.App/Filesystem/ListedItem.cs

```diff
@@ -65,6 +65,8 @@ public bool NeedsPlaceholderGlyph
             set => SetProperty(ref needsPlaceholderGlyph, value);
         }
 
+        public bool IsRecentSearch { get; set; } = false;
```

Comment:
Does this belong in ListedItem?

Reply:
Yes, I added it to differentiate recent searches from other `ListedItem`s (in order to show the correct glyph).

Reply:
This is just me wondering but I don't think ListedItem is supposed to be coupled to search or have any knowledge of search.

Reply:
Do you have anything in mind to solve this problem?
To my mind, we could add a new class (called something like `ListedSuggestion`) that inherits from `ListedItem` and expands it with the `IsSearch` property.

Reply:
Adding @d2dyno1

Reply:
Why do we even need to use/inherit from `ListedItem`? Isn't a search item a name and an icon?

Reply:
@d2dyno1